app/forms/teacher_interface/upload_form.rb

```diff
@@ -28,8 +29,9 @@ def update_model
     end
 
     def attachment_present
-      if original_attachment.blank? && translated_attachment.blank?
-        errors.add(:original_attachment, :blank)
+      errors.add(:original_attachment, :blank) if original_attachment.blank?
+      if written_in_english == "false" && translated_attachment.blank?
```

Comment:
```suggestion
      if written_in_english == false && translated_attachment.blank?
```

I’d expect the value to be converted to a bool, but if this works then no big deal!